### DISCUSSION TITLE
Search is broken?

### DISCUSSION THREAD

**Discussion by nocksock** [TanStack/router#1744] [General]
Not sure if this is on people's radar, but the search is quite unreliable:

![CleanShot 2024-06-13 at 15 13 13](https://github.com/TanStack/router/assets/22164/299914be-1b8d-4e5f-a04c-ca18de4767e1)

Had this happen a couple of times, where I couldn't find something, then googled the same + TanStack router and landed in the docs.

**Comment by schiller-manuel** [comment]
thanks for letting us know, we'll look into it
